codecov.yml

```diff
@@ -0,0 +1,22 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: auto
+        target: auto
```

Comment:
What is this set to? Will PR not be allowed to merge if codecov is under this threshold?

Reply:
threshold accepts number or auto.  if it is number say 5%, then new coverage should not go either up or below that threshold %, The numbers are only preferred if project is already released and code coverage has reached expected number. Setting auto will not enforce a min or max number instead it will only make sure that new commit should not fall below the current code coverage.

Reply:
@jmazanec15 since this the first commit to upload coverage, there is no comparison found. This will be used as base from subsequent pull request

Reply:
Makes sense thanks!